code/datums/quirks/negative.dm

```diff
@@ -219,10 +253,9 @@
 		mood.mood_modifier += 0.5
 
 /datum/quirk/hypersensitive/remove()
-	if(quirk_holder)
-		var/datum/component/mood/mood = quirk_holder.GetComponent(/datum/component/mood)
-		if(mood)
-			mood.mood_modifier -= 0.5
+	var/datum/component/mood/mood = quirk_holder.GetComponent(/datum/component/mood)
```

Comment:
weren't we against doing something like this?

Reply:
I changed the other instance of this in light step because it was doing a *= 0.6 operation and reversing that step was a pain in the ass. (I'd have to multiply by 1.666... or something and it wasn't exactly a precise or pretty solution)

This and apathetic are both easily reversed by just subtracing the exact value they added. I was going to do what I did with the light step quirk for this and apathetic too, but Mothblocks wasn't wholly satisfied with the solution and seemed to prefer the original behaviour (but begrudgingly accepted that it was difficult to cleanly reverse)

Reply:
Mood is kind of a cursed component.